docs/reference/commandline/pull.md

```diff
@@ -45,7 +45,7 @@ use `docker pull`:
     # and the un-tarred base).
     $ docker pull --all-tags centos
     # will pull all the images from the centos repository
-    $ docker pull registry.hub.docker.com/debian
+    $ docker pull docker.io/library/debian
```

Comment:
Reviewing all docs to remove any (bogus!) domain name like `registry.hub.docker.com` or `docker.io` should really be done as well...